### ISSUE TITLE
Feature Request: Add support for mydumper for exporting Databases.

### ISSUE THREAD

**Issue by planetahuevo** [sebastianfeldmann/phpbu#311]
I just discover https://github.com/maxbube/mydumper/ which is an alternative for mysqldump as they explain on this link
https://wikitech.wikimedia.org/wiki/MariaDB#Dumping_tables_with_mydumper

It is made for handling big databases and it could be a great addition to the database options. 

What do you think?

**Comment by sebastianfeldmann** [comment]
MyDumper doesn't look that wide spread in terms of usage, but it looks simple enough to add as a data source :)

**Comment by planetahuevo** [comment]
Agree, it seems is only required/useful once you reach huge databases (TBs?) , which I suppose is not that common either.

Thank you for considering it! 👍

**Comment by sebastianfeldmann** [comment]
If you are looking to backup a huge database have a look at Perconas [Xtrabackup](https://www.percona.com/doc/percona-xtrabackup/2.4/index.html)

It is a great tool du backup huge MySQL databases and iIt is used by a lot of big companies.

**Comment by planetahuevo** [comment]
Yes I know about Xtrabackup, but that is different thing as those are binary backups, with their pros and cons. 
For big databases, it is good to have both options, Xtrabackup as binary and a logical solution like MyDumper which allow easy restoration from one table, for example.
For that, mysqldump fails to provide the logical option.
